docs/agents/sematext-agent/kubernetes/operator.md

```diff
@@ -0,0 +1,73 @@
+
+
+It installs the Sematext Agent to all nodes in your cluster via a `DaemonSet` resource.
+
+## Quickstart
+
+To run the operator and its dependencies run the following command:
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/sematext/sematext-operator/master/bundle.yaml
+```
+
+## Settings
+
+This operator uses all the same options as the [Sematext Agent Helm Chart](helm.md), please take a look at all the options in the following table:
+
+|             Parameter            |            Description            |                  Default                  |
+|----------------------------------|-----------------------------------|-------------------------------------------|
+| `containerToken`                 | Sematext Container token          | `Nil` Provide your Container token        |
+| `infraToken`                     | Sematext Infra token              | `Nil` Provide your Infra token            |
+| `region`                         | Sematext region                   | `US` Sematext US or EU region             |
+| `agent.image.repository`         | The image repository              | `sematext/agent`                          |
+| `agent.image.tag`                | The image tag                     | `latest`                                  |
+| `agent.image.pullPolicy`         | Image pull policy                 | `Always`                                  |
+| `agent.service.port`             | Service port                      | `80`                                      |
+| `agent.service.type`             | Service type                      | `ClusterIP`                               |
+| `agent.resources`                | Agent resources                   | `{}`                                      |
+| `serverBaseUrl`                  | Custom Base URL                   | `Nil`                                     |
+| `logsReceiverUrl`                | Custom Logs receiver URL          | `Nil`                                     |
+| `eventsRecieverUrl`              | Custom Event receiver URL         | `Nil`                                     |
+| `serviceAccount.create`          | Create a service account          | `true`                                    |
+| `serviceAccount.name`            | Service account name              | `Nil` Defaults to chart name              |
+| `rbac.create`                    | RBAC enabled                      | `true`                                    |
+| `tolerations`                    | Tolerations                       | `[]`                                      |
+| `nodeSelector`                   | Node selector                     | `{}`                                      |
+
+For example, if you want to deploy Sematext Agent in EU region, create the following resource:
+
+```yaml
+apiVersion: sematext.com/v1alpha1
+kind: SematextAgent
+metadata:
+  name: test-sematextagent
+spec:
+  region: "EU"
+  containerToken: YOUR_CONTAINER_TOKEN
```

Comment:
Container token?  Do we still have that?

Reply:
Will wait for this before approving the PR

Reply:
We do not, deleting